spec/models/user_spec.rb

```diff
@@ -0,0 +1,63 @@
+RSpec.describe User, type: :model do
+
+  it "valid with email, user_name, password and equal confirmation" do
+    valid_user = User.new(email: 'correct_email@gmail.com',
```

Comment:
It should be in `let` or `subject`:
- https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject
- https://relishapp.com/rspec/rspec-core/v/3-10/docs/helper-methods/let-and-let#use-%60let%60-to-define-memoized-helper-method